docs/releases/4.0.txt

```diff
@@ -249,6 +249,10 @@ Models
 * :class:`~django.db.models.DurationField` now supports multiplying and
   dividing by scalar values on SQLite.
 
+* :class:`Q() <django.db.models.Q>` now supports ``__xor__`` and will be
```

Comment:
Please could you also add the new feature to the relevent documentation page(s).

Reply:
I have added documentation to the `QuerySet` and `Q` object documentations, but a review of the wording there may be needed. Thanks